src/runner.js

```diff
@@ -2,10 +2,16 @@ const createTestCafe = require("./createTestcafeGherkin");
 
 module.exports = async ({
   hostname = "localhost",
-  browsers = ["chrome:headless"],
```

Comment:
I have tested this, and actually this is not required in the `runner.js` file. Since we have a default value in `args.js`. Essentially it works the same as the default port values.